ci/.gitignore

```diff
@@ -1,5 +1,5 @@
-# SSH private key
-deploy.key
+# SSH public and private keys
+deploy.key*
```

Comment:
I don't think this should mess up repositories that already track `deploy.key.enc`

Reply:
That sounds correct to me